### ISSUE TITLE
Submit Jira tickets to create site on ihub network

### ISSUE THREAD

**Issue by allanhaggett** [allanhaggett/wp-bcgov-learning-hub#16]

**Comment by allanhaggett** [comment]
Submitted yesterday: https://govbcca.atlassian.net/servicedesk/customer/portal/2/HELP-22

**Comment by allanhaggett** [comment]
No status update as of yet; inquired with Jeremy Vernon yesterday.

**Comment by allanhaggett** [comment]
Waiting on PSA Wordpress Network PIA/STRA process to be completed before this can proceed. Could be weeks away as PSA is currently without someone in role which is suppose to process these.

**Comment by allanhaggett** [comment]
This is mostly resolved at this point. Not sure if we're sticking with Jira or what so I'm closing and will open new issue if necessary
